speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/rppUtils/RppUtils.hpp

```diff
@@ -206,8 +206,11 @@ inline void StartExecutable(const std::string& p_executablePath, const char* p_a
     // WARNING: This method of starting a process is Windows-exclusive.
     // Add a different method to run a process here if a Linux build is planned.
 
+    const char* workingDir = filesystem::path(p_executablePath).parent_path().string().c_str();
```

Comment:
This access freed memory, the string is deleted after this line is over.

Reply:
can be removed as it is not needed anymore